.github/workflows/olm_tests.yaml

```diff
@@ -134,6 +134,21 @@ jobs:
           BUNDLE_PUSH_OPT: "--tls-verify=false"
           CATALOG_PUSH_OPT: "--tls-verify=false"
 
+      - name: Install yq
+        run: |
+          sudo wget -O /usr/bin/yq https://github.com/mikefarah/yq/releases/download/v4.6.1/yq_linux_amd64
+          sudo chmod +x /usr/bin/yq
+
+      - name: Modify scorecard config
+        run: |
+          yq e 'del(.stages[].tests[] | select(.labels.test == "olm-crds-have-resources-test"))' -i bundle/tests/scorecard/config.yaml
+
+      - name: Scorecard check
```

Comment:
Thinking whether it would have sense to move this test into separate workflow, as it is not related to OLM install or upgrade testing.

Reply:
I agree that moving it to a separate workflow makes sense

Reply:
it does, but I had it here to try and reduce time, moving it to its own workflow means we'd have to build it again. 
no issue either way really, I just hate doing things twice when there is no need